tests/test_probability_premature_death.py

```diff
@@ -0,0 +1,55 @@
+import datetime
+import os
+from pathlib import Path
+
+from tlo.analysis.life_expectancy import get_probability_of_premature_death
+from tlo.analysis.utils import extract_results
+
+target_period=(datetime.date(2010, 1, 1), datetime.date(2080, 12, 31))
+AGE_BEFORE_WHICH_DEATH_IS_DEFINED_AS_PREMATURE = 70
+
+def test_probability_premature_death(tmpdir):
```

Comment:
I would add to the `test_life_expectancy.py` file as that file already exists and it seems reasonable for to 'house' test for the two function in the `life_expectnancy.py` file

Reply:
Grand! Have done